src/config/paper/help.yml

```diff
@@ -0,0 +1,110 @@
+general-topics:
+  <topic name>:
+    header:
+      message: A custom help "topic" to show when the user types "`/help <topic>`"
+    shortText:
+      description: |
+        The first line of the help entry, used as a summary in index pages.
+
+        This is formatted with legacy "&" style formatting
+      default: '""'
+    fullText:
+      description: |
+        The remaining lines of this help entry.
+
+        This is formatted with legacy "&" style formatting
+      default: '""'
+    permission:
+      description: |
+        The permission required to view this help entry.
+
+        If this is empty/not set, the entry does not require a permission
+      default: '""'
+
+index-topics:
+  <topic name>:
+    header:
+      message: |
+        An composite help topic index.
+
+        If the topic name is the special value "Default",
+        it will override the automatically generated index used for /help with no arguments
+    shortText:
+      description: |
+        The short text used in other index topics
+
+        This is not displayed when showing this topic itself.
+
+        This is formatted with legacy "&" style formatting
+      default: '""'
+    preamble:
+      description: |
+        Information to show before the index
+
+        This is formatted with legacy "&" style formatting
+      default: '""'
+    permission:
+      description: |
+        A custom permission required to view this help entry.
+
+        If this is empty/not set, the entry does not require a permission
+
+        This can be formatted with legacy "&" style formatting????? (but don't use it)
```

Comment:
```suggestion
        This can be formatted with legacy "&" style formatting (?)
```

Reply:
I'm wondering whether to include this mention at all tbh.

I suspect its a bug, so it might be better to pr this behavior out, since its confusing and shouldn't be a thing

![image](https://github.com/user-attachments/assets/fd545a7c-b46a-4874-b83d-f5d77f397a64)

Reply:
sounds good :+1: